### PR TITLE
Gemma 2b configs

### PR DESCRIPTION
**Pull request by rasbt**
This is a really tough one. I tried a lot of different settings and this was the best I could find for LoRA and QLoRA. 

- A val loss of 1.4 seems high, but it's coming down from >15, so it's a lot improvement
- Anything else I tried results in more overfitting
- Interestingly, I looked at and tried the Axolotl configs in Axolotl, and their loss even seems to increase to like 70, not converging at all
- The full finetuning is in the works, this is even tougher; for some reason gemma-2b is really hard to train; I will probably move on to other models now because it's been quite the time sink
- at least we see that QLoRA saves some memory over LoRA here.

### CODE REVIEW

**Review by awaelchli — APPROVED** [review]

File: config_hub/finetune/gemma-2b/lora.yaml
Comment (carmocca) [inline config_hub/finetune/gemma-2b/lora.yaml:1]: Remember to add the test for the config!

**Comment by rasbt** [comment]
Will add a new PR including the full finetuning shortly.
